### PR TITLE
Able to use sauce username/accesskey to run tests vslt/vslt-114

### PR DESCRIPTION
**Pull request by luisVelasquezSauce**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Change Log
 
+## 0.11.11
```
Comment (loyalchow) [inline CHANGELOG.md:3]: Update minor version

File: README.md [hunk 2]
```diff
@@ -42,6 +42,30 @@ module.exports = {
...
 
+#### Using Sauce Credentials to authenticate
+```javascript
+module.exports = {
+  // full repository name for your project:
+  projectRepo: 'user/my-project-repo',
+
+  // this example assumes Environment Variables listed below exist on your system:
+  username: process.env.SAUCE_USERNAME,
+  accessKey: process.env.SAUCE_USERNAME,
```
Comment (loyalchow) [inline README.md:53]: environment variable should be `SAUCE_ACCESS_KEY`

File: README.md [hunk 3]
```diff
@@ -20,11 +20,11 @@ Then add a configuration file to your project root. Here is an example:
 
 ### screener.config.js
 
+#### Using API Key to authenticate
```
Comment (loyalchow) [inline README.md:23]: Suggest adding "Screener" to be a bit more explicit: `Using Screener's API Key to authenticate`

File: src/runner.js [hunk 4]
```diff
@@ -149,7 +156,7 @@ exports.run = function(config) {
       if (tunnelHost) {
         config.states = transformToTunnelHost(config.states, config.tunnel.host, tunnelHost);
       }
-      var payload = omit(config, ['apiKey', 'resolution', 'resolutions', 'includeRules', 'excludeRules', 'tunnel', 'failureExitCode', 'sauce.launchSauceConnect']);
+      var payload = omit(config, ['apiKey', 'username', 'accessKey', 'resolution', 'resolutions', 'includeRules', 'excludeRules', 'tunnel', 'failureExitCode', 'sauce.launchSauceConnect']);
```
Comment (loyalchow) [inline src/runner.js:159]: I think we may need to keep this. Currently the `sauce` option gets passed to test-runner eventually with sauce creds. Or we keep this the way it is, and let's have a separate ticket to think through how to best handle that scenario

File: test/validate.spec.js [hunk 5]
```diff
@@ -133,10 +133,18 @@ describe('screener-runner/src/validate', function() {
         });
     });
 
-    it('should throw error when no apiKey', function(done) {
-      Validate.runnerConfig({})
+    it('should throw error when no apiKey or username', function(done) {
```
Comment (loyalchow) [inline test/validate.spec.js:136]: Can you add a test for the xor? That both apiKey and username/accessKey can't exist at the same time

**Review by loyalchow — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (luisVelasquezSauce) [reply README.md]: Done

File: test/validate.spec.js (hunk 5 above)
Comment (luisVelasquezSauce) [reply test/validate.spec.js:144]: Done

File: src/runner.js (hunk 4 above)
Comment (luisVelasquezSauce) [reply src/runner.js:159]: Sorry can you clarify this a bit? Thanks @loyalchow

**Comment by loyalchow** [comment]
@luisVelasquezSauce LGTM 👍 - as discussed, please don't merge yet till App updates are deployed

**Review by loyalchow — APPROVED** [review]

**Pull request closed without merging**